lib/ansible/plugins/doc_fragments/checkpoint_facts.py

```diff
@@ -0,0 +1,16 @@
+# -*- coding: utf-8 -*-
```

Comment:
@chkp-orso if the facts modules are moving to lookup plugin, so is it required to have this doc_fragments for `checkpoint_facts`

Reply:
@justjais 
1. The error is complete, so I leave it as it is
2. there is `checkpoint_facts` in doc_fragments, and there is the line u wrote..